api/src/main/java/tennisclub/dto/CourtCreateDto.java

```diff
@@ -0,0 +1,64 @@
+package tennisclub.dto;
+
+import tennisclub.enums.CourtType;
+
+public class CourtCreateDto {
```

Comment:
I've just also noticed one thing from the reference implementation - the XXXCreateDTOs have annotations for validating attributes. So in the case of `Court`, we don't want to have nullable name _(therefore there should be a `@NotNull` annotation; also we don't probably want an empty string as a name, so there could be a `@Size` annotation)_ 😊 I'm sorry I didn't notice it earlier 🙏

Reply:
Again, thank you for noticing! Will fix.

Reply:
@Fuyunori, done.

Reply:
Thank you very much! I think you're just missing also `@NotNull` annotation above the name attribute, and with that it's perfect! 😃 well done and good job :3